### PR TITLE
FIX(Bmp-210): automatically set to latest version with finalname

### PR DESCRIPTION
**Pull request by FilippoRezzonico**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM alpine AS build
 
 ARG COMMIT_SHA=<not-specified>
-ARG BUILD_FILE_NAME=%CUSTOM_PLUGIN_SERVICE_NAME%-0.1.0-SNAPSHOT
+ARG BUILD_FILE_NAME=%CUSTOM_PLUGIN_SERVICE_NAME%-latest-SNAPSHOT
```
Comment (fredmaggiowski) [inline Dockerfile]:
```suggestion
ARG BUILD_FILE_NAME=%CUSTOM_PLUGIN_SERVICE_NAME%
```

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
     <groupId>%CUSTOM_PLUGIN_PROJECT_NAMESPACE%</groupId>
     <artifactId>%CUSTOM_PLUGIN_SERVICE_NAME%</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
```
Comment (fredmaggiowski) [inline pom.xml]: The version must be specified

File: pom.xml [hunk 3]
```diff
@@ -39,6 +38,9 @@
     </properties>
 
     <build>
+
+    <finalName>${project.artifactId}</finalName>
```
Comment (fredmaggiowski) [inline pom.xml:42]: Please indent it properly and remove useless whitelines above and below

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (FilippoRezzonico) [reply pom.xml:42]: removed whitelines :)

File: pom.xml (hunk 2 above)
Comment (FilippoRezzonico) [reply pom.xml]: re-added version

File: Dockerfile (hunk 1 above)
Comment (FilippoRezzonico) [reply Dockerfile]: applied suggestion

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**